apps/src/code-studio/pd/workshop_dashboard/workshop.jsx

```diff
@@ -166,6 +166,31 @@ export class Workshop extends React.Component {
     });
   };
 
+  handleMoveEnrollment = (destinationWorkshopId, selectedEnrollments) => {
+    const enrollmentIds = selectedEnrollments.map(enrollment => {
+      return enrollment.id;
+    });
+    const urlParams = `enrollment_ids[]=${enrollmentIds.join(
+      '&enrollment_ids[]='
+    )}`;
```

Comment:
Why do we put the enrollment ids both in the queryparams and in the body of the POST request?

Reply:
not needed in the body! thanks!